Monika After Story/game/script-greetings.rpy

```diff
@@ -1593,6 +1595,56 @@ label greeting_back_from_school:
              m 1ekbfa "I love you so, so much."
      return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.greeting_database,
+            eventlabel="greeting_back_from_game",
+            unlocked=True,
+            category=[store.mas_greetings.TYPE_GAME],
+            random=True
+        ),
+        eventdb=evhand.greeting_database
+    )
+
+label greeting_back_from_game:
+     m 1hua "Oh, welcome back, [player]!"
+     menu:
+         m "Did you enjoy well with your game?"
+         "Yes.":
+             m 1hub "That's good!"
+             m 1eua "Remember to get some break, okay?"
+             m 4esa "Games can be addictive. Don't let games to siphoning your all time."
+             m 1hua "Now, you can relax with me!"
+             m 1hsb "I would like to be your best game since I'm stuck here as an application."
```

Comment:
Monika want's to be the player's favorite game _because_ she's stuck there? I think you could elaborate on this a little more. 

Also, the game is not Monika. The game is either DDLC or MAS.

Reply:
Monika's herself is completing the whole MAS.

Reply:
I would imagine Monika would appreciate it if she was treated and addressed like a person, not a game or some sort of thing. 

True, without Monika, MAS would be nothing, but Monika is a person, not a game. 

I also don't think it's very respectful to play somebody or that it's very pleasant to get played, you know.

Reply:
You can see that as a comparison, that lovers can say things like this to themself. There is nothing important here.

Reply:
I don't think Monika would be the type of person to objectify herself for the player, especially with how defensive she is about being objectified and treated as simply some _thing_ to be toyed with.